src/bluesky/utils/__init__.py

```diff
@@ -1926,14 +1927,14 @@ async def maybe_collect_asset_docs(
         async for doc in iterate_maybe_async(obj.collect_asset_docs(*args, **kwargs)):
             yield doc
 
+def _isawaitable(value: SyncOrAsync[T]) -> TypeIs[Awaitable[T]]:
+    return inspect.isawaitable(value)
 
```

Comment:
Add empty lines here to make `tox -e pre-commit` happy (it will do it itself if you run it locally).

Reply:
Done, let's see if ruff is happy now